rust/src/measurements/laplace/float/make_scalar_float_laplace_cks20.tex

```diff
@@ -0,0 +1,115 @@
+\documentclass{article}
+\input{../../../lib.sty}
+
+
+
+\title{\texttt{fn make\_scalar\_float\_laplace\_cks20}}
+\author{Michael Shoemate}
+\begin{document}
+
+\maketitle
+
+\contrib
+
+Proves soundness of \rustdoc{measurements/fn}{make\_scalar\_float\_laplace\_cks20} in \asOfCommit{mod.rs}{f5bb719}.
+The function on the resulting measurement takes in a data set \texttt{x} (a single float), 
+and returns a sample from the Discrete Laplace Distribution centered at \texttt{x}, with a fixed noise scale.
+
+\subsection*{PR History}
+\begin{itemize}
+    \item \vettingPR{490}
+\end{itemize}
+
+\section{Hoare Triple}
+
+\subsection*{Preconditions}
+\begin{itemize}
+    \item Variable \texttt{input\_domain}, of type \texttt{AtomDomain<T>}
+    \item Variable \texttt{input\_metric}, of type \texttt{AbsoluteDistance<T>}
+    \item Variable \texttt{scale}, of type \texttt{QO}
+    \item Variable \texttt{k}, of type \texttt{Option<i32>}
+    \item Type \texttt{T} must have trait \rustdoc{traits/trait}{Float} and \rustdoc{traits/trait}{CastInternalRational}
+    \item Type \texttt{i32} must be constructable from the bit representation of \texttt{T} (used to calibrate relaxation)
+\end{itemize}
+
+\subsection*{Pseudocode}
+\lstinputlisting[language=Python,firstline=2,escapechar=|]{./pseudocode/make_scalar_float_laplace_cks20.py}
+
+\subsection*{Postcondition}
+
+\validMeasurement{\texttt{(input\_domain, input\_metric, scale, k, T)}}{\\\texttt{make\_scalar\_float\_laplace\_cks20}}
+
+\section{Proof}
+
+\begin{proof} 
+\textbf{(Privacy guarantee.)} 
+    
+\begin{tcolorbox}
+    The proof assumes the following lemma.
+    \begin{lemma}
```

Comment:
I would also explicitly state the lemmata for these functions. These are crucial functions for this method, so having them defined, at least in terms of their functionality, would be useful.

Reply:
Adjusted these to be links.